components/general/NetPyNECoordsRange.js

```diff
@@ -87,7 +87,9 @@ export default class NetPyNECoordsRange extends Component {
             <PythonControlledAdapterComponent
               model={path}
               convertToPython={(state) => {
-                if (state.min != undefined && state.max != undefined) {
+                if(state[state.lastUpdated].toString().endsWith("."))
+                  return undefined;
+                if ((!isNaN(parseFloat(state.min))) && (!isNaN(parseFloat(state.max)))) {
```

Comment:
Probably better to have everything in a single 'if' that checks the format is correct:
```
                if (!state[state.lastUpdated].toString().endsWith(".") && 
                ((!isNaN(parseFloat(state.min))) && (!isNaN(parseFloat(state.max))))) {
```